plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -309,8 +309,8 @@ computeBaseUrl(char* serverParam, int defaultPort, sbool useHttps, uchar **baseU
 		host = serverParam + strlen(SCHEME_HTTPS);
 	else
 		r = useHttps ?
-			es_addBuf(&urlBuf, SCHEME_HTTPS, sizeof(SCHEME_HTTPS)-1) : 
-			es_addBuf(&urlBuf, SCHEME_HTTP, sizeof(SCHEME_HTTP)-1);
+		    es_addBuf(&urlBuf, SCHEME_HTTPS, sizeof(SCHEME_HTTPS)-1) : 
```

Comment:
The rule, unfortunately I think not yet written anywhere, is that if we indent, we ident by one TAB. So IMHO this should remain as it was.